example/linux/systemd/Talawa-api.sh

```
@@ -0,0 +1,52 @@
+# Define the path to the tsx executable dynamically
+TSX_PATH=$(which tsx)
+
+# Define paths for development and production
+DEV_PATH="src/index.ts"
+PROD_PATH="dist/index.js"
+
+# Change to the project directory .give your path
+cd /home/purnendu/Development/talawa-api
+
+# Load environment variables from .env file
+export $(grep -v '^#' .env | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)